bin/cli-meli-accounts/index.js

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env node
+const envPath = require('path').resolve(__dirname, '.env')
+require('dotenv').config({path: envPath})
+
+const program = require('commander')
+
+program
+  .version('0.1.0')
+  .description('Interactive MercadoLibre user accounts CLI management.')
+  .option('-u, --user <nickname>', 'Run using specified Account nickname for Meli API requests.')
+  .parse(process.argv)
+
+if (!program.user) {
+  console.log('Please specify --user <nickname> option.')
+  program.outputHelp()
+  process.exit()
+}
+
+const db = require('../../config/db')
+const Account = require('../../model/account')
+const prompt = require('./prompt')
+
+const cliLoginFlow = require('./cli-login-flow')
+const createMeliTestAccount = require('./create-meli-test-account')
+
+const devAccountNickname = program.user
+
+async function doLoginFlow() {
+  let accountTokens
+  try {
+    accountTokens = await cliLoginFlow.run()
+  } catch (error) {
+    throw new Error(`Ups, could not authenticate: ${error.message || error}`)
+  }
+  console.log('Logged in!')
+  return accountTokens
+}
+
+async function generateTestAccount() {
+  let testAccount
+  try {
+    testAccount = await createMeliTestAccount(devAccountNickname)
+  } catch (error) {
+    // TODO if error is lack of dev account, retry? suggest a different client id?
+    throw new Error(`Whoops, could not create a test account: ${error.message || error.data || error}`)
+  }
+  console.log('Test account is: ', testAccount)
+}
+
+async function registerAccount({profile, tokens}) {
+  let account
+  try {
+    account = await Account.register(profile, tokens)
+  } catch (error) {
+    throw new Error('Problem registering account: ' + (error.message || error))
+  }
+  const verbMsg = account.isNewAccount() ? 'Registered new' : 'Updated existing'
+  console.log(`${verbMsg} ${account.isTestAccount ? 'test' : ''}account '${account.nickname}' succesfully.`)
+}
+
+const options = {
+  newTestAccount: async () => {
+    console.log('Creating test account...')
+    try {
+      await generateTestAccount()
+      const loggedUser = await doLoginFlow()
+      await registerAccount(loggedUser)
+    } catch (error) {
+      console.error(error.message || error)
+    }
+  },
+  existingAccount: async () => {
+    console.log('Please log in with an existing account.')
+    try {
+      const loggedUser = await doLoginFlow()
+      await registerAccount(loggedUser)
+    } catch (error) {
+      console.error(error.message || error)
+    }
+  },
+  exit: () => {
+    console.log('Bye!')
+  }
+}
```

Comment:
Use a Map<string, Function>, better performance + conceptually more appropriate

Reply:
it's a good suggestion, but I think for this case it's an overkill. The object we need is simple, static, we don't need to iterate it, and "access time" is not significant at all to justify an optimization. The Map structure is much more powerful for these use cases but it is none of ours. In the other hand, the base Object structure is much more simple and straightforward. 
Here a more deeper analysis and comparison: https://medium.com/front-end-hacking/es6-map-vs-object-what-and-when-b80621932373